### PR TITLE
test(bench): Add Django benchmark

### PR DESCRIPTION
**Pull request by kgriffs**

### CODE REVIEW

File: falcon/bench/bench.py [hunk 1]
```diff
@@ -254,11 +254,12 @@ def run(frameworks, trials, iterations, stat_memory):
 def main():
     frameworks = [
         'bottle',
+        'django',
         'falcon',
         'falcon-ext',
         'flask',
         'pecan',
-        'werkzeug'
+        'werkzeug',
```
Comment (ueg1990) [inline falcon/bench/bench.py]: any reason for the trailing comma? I know it still works

Comment (kgriffs) [inline falcon/bench/bench.py]: Oh, it's just an old stylistic habit of mine. It avoids problems when you rearrange lines or whatnot and have to remember to add the comma at that point.

Comment (ueg1990) [inline falcon/bench/bench.py]: ah ok :)

File: falcon/bench/dj/dj/settings.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.11/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 't6gj%z0ee)*xmdoqhm4^er60=s^1g_vs7y!dl2vve4-1u9^+ps'
```
Comment (jmvrbanac) [inline falcon/bench/dj/dj/settings.py]: :fire: lol. Considering this is just a perf test it's fine. It still does give me the shivers though.

Comment (AndrewBurdyug) [inline falcon/bench/dj/dj/settings.py]: Maybe it is better to generate it on fly: 

```python 
from django.core.management import utils; 
utils.get_random_secret_key()
```
in similar way django cmd `./manage.py startapp some_app` generate *SECRET_KEY* option...

Comment (jmvrbanac) [inline falcon/bench/dj/dj/settings.py]: Awesome! Thanks for the info. Corrected in #1088

Comment (AndrewBurdyug) [inline falcon/bench/dj/dj/settings.py]: :)

**Pull request merged**